Tests/StencilSwiftKitTests/JSFiltersTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+import StencilSwiftKit
+
+class JSFiltersTests: XCTestCase {
+  
+  func testJSFilter() {
```

Comment:
Please rewrite the tests to be similar to the other node/filter tests. This means to write tests with a few parts:
- given a list of tokens, parses a correct structure of nodes/filters and such
- given a structure of nodes and a context, renders the correct output
- some tests to verify that parsing/rendering correctly fails for invalid data.